travis/trusty/post/cargo-sweep.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
 # Install cargo-sweep. If installed, cargo failure will be ignored.
+# TODO: parse ~/.cargo/.crates.toml to determine version
+#       maybe use cargo-update library for that
 cargo install cargo-sweep --vers 0.4.1 || true
 
 # cargo-sweep produces a lot of output
-cargo sweep . -t 30 > /dev/null
+cargo sweep --toolchains $TRAVIS_RUST_VERSION > /dev/null
```

Comment:
Is it possible that TRAVIS_RUST_VERSION will be different from our pinned rustc nightly version? If so, is this the desired behaviour?

Reply:
The desired behavior is to run `cargo sweep --toolchains nightly-2019-03-20`, so everything created by other toolchains will be purged from cache (also, everything older than 30 days made by `nightly-2019-03-20`, but that's default behavior).

I hope `TRAVIS_RUST_VERSION` is equal to the pinned version, I will add `echo` just to be sure (cargo-sweep returns 0 even on error, I will report that as a bug)

Reply:
I have just seen the print output, I'm convinced (:

```bash
$ travis/trusty/post/cargo-sweep.sh
    Updating crates.io index
  Installing cargo-sweep v0.4.1
error: binary `cargo-sweep` already exists in destination as part of `cargo-sweep v0.4.1`
Add --force to overwrite
Rust toolchain version: nightly-2019-03-20
The command "travis/trusty/post/cargo-sweep.sh" exited with 0.
```
I wonder how it works. I was almost sure that `TRAVIS_RUST_VERSION` will be some generic rustc version like the latest nightly.